### PR TITLE
[BACKLOG-6634] Compatibility with 6.1

### PR DESCRIPTION
**Pull request by nantunes**

### CODE REVIEW

File: Assemblies/PDI/src/main/feature/feature.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <bundle dependency="false">pentaho-webjars:mvn:org.webjars.bower/angular-ui-router/0.2.15</bundle>
     <bundle dependency="false">pentaho-webjars:mvn:org.webjars.bower/ui-router-extras/0.0.13</bundle>
     <bundle dependency="false">pentaho-webjars:mvn:org.webjars.bower/angular-ui-router.stateHelper/1.3.1</bundle>
-    <bundle dependency="false">pentaho-platform-plugin-mvn:pentaho/common-ui/6.1-20151213.180953-705/zip</bundle>
```
Comment (graimundo) [inline Assemblies/PDI/src/main/feature/feature.xml]: Change this to a POM property named dependency.common-ui.revision
Change the current POM property https://github.com/webdetails/det/blob/master/pom.xml#L61 to dependency.common-ui.revision

File: Assemblies/Platform/src/main/feature/feature.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <details>Pentaho Data Explorer Tool for BA Assembly</details>
 
     <bundle dependency="true">mvn:pentaho/pentaho-requirejs-osgi-manager/6.1-SNAPSHOT</bundle>
-    <bundle dependency="false">pentaho-platform-plugin-mvn:pentaho/common-ui/6.1-20151213.180953-705/zip</bundle>
+    <bundle dependency="false">pentaho-platform-plugin-mvn:pentaho/common-ui/6.1-SNAPSHOT/zip</bundle>
```
Comment (graimundo) [inline Assemblies/Platform/src/main/feature/feature.xml]: Change version to POM property dependency.common-ui.revision

**Comment by graimundo** [comment]
While you're at it update common-ui version in the following places:
https://github.com/webdetails/det/blob/master/Implementation/Platform/pom.xml#L61
https://github.com/webdetails/det/blob/master/Implementation/PDI/pom.xml#L78
https://github.com/webdetails/det/blob/master/Implementation/Core/pom.xml#L117

Alternatively, set the common-ui version in the dependency management of the parent POM and inherit it in the child POMs.

Other than that, good to go.

**Comment by nantunes** [comment]
@graimundo please recheck.

**Pull request merged**